templates/ci/python-django.yml

```diff
@@ -0,0 +1,73 @@
+# Python Django
+# Test a Django app on multiple versions of Python.
+# https://aka.ms/yaml
+
+## NOTE! You need to set the projectRoot in both phases (until VSTS supports YAML anchors).
```

Comment:
@DavidStaheli this is kind of inelegant, but we don't have a better solution right now. Are you OK with it?